### PR TITLE
Feature/verify client version

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -139,10 +139,10 @@ jobs:
           ASAN_OPTIONS: fast_unwind_on_malloc=1
         run: |
           pushd build
-          ctest -j $(($(nproc)/2+1)) --timeout 400 --output-on-failure
+          ctest -j $(nproc) --timeout 10000 --output-on-failure
```
Comment (Aiee) [inline .github/workflows/pull_request.yml:142]: Why change the number of cores?

File: include/common/graph/Response.h [hunk 2]
```diff
@@ -400,4 +400,14 @@ struct ExecutionResponse {
...
 };
 
+struct VerifyClientVersionResp {
+  ErrorCode errorCode{ErrorCode::SUCCEEDED};
+  std::unique_ptr<std::string> errorMsg{nullptr};
+};
+
+struct VerifyClientVersionReq {
+  // TODO initialize when build
+  std::string version{"2.6.0"};
```
Comment (Aiee) [inline include/common/graph/Response.h:410]: 2.6.1 should also be compatible

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (Shylock-Hg) [reply .github/workflows/pull_request.yml:142]: Speed up.

File: include/common/graph/Response.h (hunk 2 above)
Comment (Shylock-Hg) [reply include/common/graph/Response.h:410]: It's the version of client.

**Review by Aiee — APPROVED** [review]

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (Aiee) [reply .github/workflows/pull_request.yml:142]: This may cause other PRs' actions to fail.

Comment (Shylock-Hg) [reply .github/workflows/pull_request.yml:142]: Never happen.

**Pull request merged**